### DISCUSSION TITLE
Populate a repeater with a non-related collection.

### DISCUSSION THREAD

**Discussion by StaffCollab** [filamentphp/filament#6925] [Help] [answered]
I'm troubling around getting a repeater to pre-populate on a create form.

I had issues with getting relationships to work as this is a create form which doesn't have relationships yet, (although it should have as I'm using it in a relations manager, I guess it is not supported because it is not yet persisted into the DB)

The question I have now is if I can do my own eloquent query to get some collection and then populate the repeater with that unrelated collection, can we do something like that?

I think we can do it with regular tall forms. The question is about filament admin.

**Comment by danharrin** [comment, marked as answer]
Duplicate of #6920?

**Reply by StaffCollab** [reply]
You're right. While I was working on a workaround, I tried this approach and thought it would be best to ask about it.
Sorry for the duplication.
